.github/workflows/codeql-analysis.yml

```diff
@@ -1,72 +0,0 @@
-# For most projects, this workflow file will not need changing; you simply need
```

Comment:
don't you think that is a good idea to have it?
or later are you going to add the sonarcube?

Reply:
Well, for now, there is codeql/sonarcube analysis, so I don't see the point of that job

Reply:
We may introduce it later.